test_image.sh

```diff
@@ -62,6 +63,7 @@ function main {
 	test_docker_image_hotfix_installed
 	test_docker_image_scripts_1
 	test_docker_image_scripts_2
+	test_docker_image_patching_tool_update
```

Comment:
If there is no dependency between the tests, this should be in ABC order as well.

Reply:
Fixed.